src/components/EmailForm/index.jsx

```diff
@@ -0,0 +1,77 @@
+import { useState } from "react"
+import { Link } from "react-router-dom"
+
+function EmailForm(){
+    const [email,setEmail]=useState('')
+    const [subject,setSubject]=useState('')
+    const [message,setMessage]=useState('')
+    const handleSubmit=(e)=> {
+        e.preventDefault()
+        console.log(email)
+        console.log(subject)
+        console.log(message)
+        getData()
+    }
+  function getData (){
+    fetch('https://email-client-api.dev.io-academy.uk/emails', {
+  method: 'POST',
+  headers: {
+    "content-type": "application/json"
+  },
+  body:JSON.stringify({
+    name: "username",
+     email:email,
+      subject:subject,
+      body:message
+      })
+})
+  .then(res => res.json())
+  .then(data => {
+    console.log(data)
```

Comment:
This data that comes back will contain a message that you can display as a success/failure message.

If the email fails to send for whatever reason (like sending it invalid email addresses) it will give you an appropriate message to display to the users